website/docs/docs/build/ref-metrics-in-filters.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Metrics as dimensions with metric filters"
+id: "ref-metrics-in-filters"
+description: "Add metrics as dimensions to your metric filters to create more complex metrics and gain more insights."
+sidebar_label: "Metrics as dimensions"
+---
+
+[Metrics](/docs/build/metrics-overview) provide users with valuable insights into their data, such as the number of active users. They also are used to track performance, trend, and help businesses make important decisions. [Dimensions](/docs/build/dimensions), on the other hand, are attributes that help categorize data, such as user type or number of orders placed by a customer.
+
+To make informed business decisions, some metrics need the value of another metric as part of the metric definition, leading us to "metrics as dimensions".
+
+This document explains how you can use metrics as dimensions with metric filters, enabling you to create more complex metrics and gain more insights.
+
+## Use cases
```

Comment:
Just a note that all of these use cases are not currently supported. Currently we only allow one entity in the group by for the metric filter. The next iteration of this feature will enable these use cases - users will be able to pass multiple entities OR dimensions in the group by for the metric filter. Should we remove these use cases from the docs until they are supported? Or maybe add them to the bottom of the page with a note saying this is coming soon?

Reply:
(this only applies to the `Use cases` section prior to the `Example` section)